packages/x-charts/src/PieChart/PieArc.tsx

```diff
@@ -66,9 +67,9 @@ const PieArcRoot = styled('path', {
   name: 'MuiPieArc',
   slot: 'Root',
   overridesResolver: (_, styles) => styles.arc, // FIXME: Inconsistent naming with slot
-})<{ ownerState: PieArcOwnerState }>(({ theme }) => ({
+})<{ ownerState: PieArcOwnerState }>(({ ownerState, theme }) => ({
   // Got to move stroke to an element prop instead of style.
-  stroke: (theme.vars || theme).palette.background.paper,
+  stroke: ownerState.stroke ?? (theme.vars || theme).palette.background.paper,
```

Comment:
Do we need to put that in the `ownerState`? The `stroke` property could be a prop

Reply:
I wasn't sure why it was defined in the styles to begin with, happy to move it outside of the styles 👌